guardrails/telemetry/runner_tracing.py

```diff
@@ -265,6 +265,11 @@ def trace_call_wrapper(*args, **kwargs):
             ) as call_span:
                 try:
                     response = fn(*args, **kwargs)
+                    if isinstance(response, LLMResponse) and (
```

Comment:
thanks @zsimjee this doesnt make sense to do in a streaming context because itll cause the stream to be read to serialize

Reply:
totally can track attributes later